src/dxtbx/format/FormatNXmx.py

```diff
@@ -60,8 +60,8 @@ def _start(self):
     def _get_nxmx(self, fh: h5py.File):
         return dxtbx.nexus.nxmx.NXmx(fh)
 
-    def _beam(self, index=None):
-        return self._beam_model
+    def _beam(self, index: int = 0):
+        return self._beam_factory(index=index)
```

Comment:
```suggestion
        return self._beam_factory(index=index or 0)
```
I'm seeing a stack trace of:
```
  File "dxtbx/format/FormatMultiImage.py", line 293, in get_imageset
    beam = format_instance.get_beam()
  File "dxtbx/format/FormatNexus.py", line 104, in get_beam
    return self._beam(index)
  File "dxtbx/format/FormatNXmx.py", line 64, in _beam
    return self._beam_factory(index=index)
  File "dxtbx/src/dxtbx/nexus/__init__.py", line 72, in __call__
    assert len(self.incident_wavelength) > index, len(self.incident_wavelength)
```
e.g. `FormatMultiImage` calls `FormatNexus` `get_beam()` with the default:
https://github.com/cctbx/dxtbx/blob/cd4eb058b0a0f64397ec5053dc33e024dd783779/src/dxtbx/format/FormatNexus.py#L103-L104

Which means that this function gets called with explicit `None` even though it has a default `=0`.